src/main/java/athena/Parser.java

```diff
@@ -100,7 +98,7 @@ public static Commands parse(String input) {
 
             String importance;
             if (iPosition == -1) {
-                importance = "medium";
+                importance = "MEDIUM";
```

Comment:
It may be better to leave this in lowercase, as this is what our user guide says. We can convert this into an enum in our next version?